src/app/ApplicationTemplate.UWP/Views/Styles/Controls/Button.xaml

```diff
@@ -254,6 +254,95 @@
 		</Setter>
 	</Style>
 
+	<Style x:Key="SettingsButtonStyle"
```

Comment:
Style names should describe **what** they do, **how** they look, etc.
They should **not** be about **where** they are expected to be used.